modules/33acrossIdSystem.js

```diff
@@ -97,6 +98,11 @@ function calculateQueryStringParams(pid, gdprConsentData, enabledStorageTypes) {
     params.tp = encodeURIComponent(tp);
   }
 
+  const hem = getStoredValue(STORAGE_HEM_KEY, enabledStorageTypes);
```

Comment:
what if a publisher wanted to pass this on your config params?

Reply:
Hi @patmmccann 

that's a good point, I think we could easily implement this functionality and add it to the PR... working on it.

Reply:
I've added the option to pass hashed email via configuration param.